encryptor/config/encryptionSettings.go

```diff
@@ -230,38 +230,43 @@ func (s *BasicColumnEncryptionSetting) Init() (err error) {
 	if s.ReEncryptToAcraBlock != nil && *s.ReEncryptToAcraBlock {
 		s.settingMask |= SettingReEncryptionFlag
 	}
+	var tokenType common.TokenType
+	var ok bool
 	if s.TokenType != "" || s.Tokenized {
-		tokenType, ok := tokenTypeNames[s.TokenType]
+		tokenType, ok = tokenTypeNames[s.TokenType]
```

Comment:
Here we define `ok` in scope of the entire function, though it is used only in scope of this `if` statement. And then, the `ok` is overwritten on the 297th line. For the reader this is confusing, because my first assumption was that the result of this variable is used somewhere below, and I scroll up and down a couple of times to see where it is used.

Reply:
there is only `tokenType` used in the block below. But without pre-declaring `ok` variable `tokenType` in the `if` block will be different and don't  re-use tokenType from outside block.